R/checks.R

```diff
@@ -1071,6 +1080,30 @@ checkSingleColon <- function(Rdir, avail_pkgs = character(0L)) {
     msg_sc <- unlist(msg_sc)
 }
 
+checkPasteInSignaler <- function(Rdir) {
+    rfiles <- dir(Rdir, pattern = "\\.[Rr]$", full.names = TRUE)
+    names(rfiles) <- basename(rfiles)
+    paste_sig <- lapply(rfiles, function(rfile) {
```

Comment:
Maybe make this anonymous function a named function, to help with readability and testing? Otherwise this looks like a great addition!

Reply:
https://github.com/Bioconductor/BiocCheck/pull/111/files#diff-6cd987920443d42098a4963d43b2e0efec9fd16f9198d66705e6d95094e3b87aR1083-R1100